helpers/index.js

```diff
@@ -11,6 +12,10 @@
    * var app = express();
    * app.use(helpers.errorHandler);
    * */
+
+  var gauge = new client.Gauge('nodejs_errors_total', 'This is a counter of all the exceptions handled.');
```

Comment:
Yeah, actually, I don't think you need this metric at all. It can be calculated from the histogram metric (if it contains a status code).